hardware_interface/src/mock_components/generic_system.cpp

```diff
@@ -723,11 +723,11 @@ void GenericSystem::initialize_storage_vectors(
           if (it2 != component.parameters.end())
           {
             states[index][i] = hardware_interface::stod(it2->second);
-            print_hint = true;
+            print_hint = false;
```

Comment:
```suggestion
            print_hint = true;
```

Reply:
we want to print the hint if the old parameter is used

Reply:
understood.